packages/app/src/vis-packs/core/matrix/utils.ts

```diff
@@ -80,6 +82,20 @@ export function getFormatter(
   return (val: string) => val.toString();
 }
 
+export function getCsvFormatter<T extends PrintableType>(
+  type: T,
+): (val: ScalarValue<T>) => string;
+
+export function getCsvFormatter<T extends PrintableType>(
+  type: T,
+): ValueFormatter<PrintableType> {
+  if (isComplexType(type)) {
+    return createComplexFormatter((val) => val.toString());
+  }
+
+  return (val: string | number | bigint | boolean) => val.toString();
```

Comment:
I chose to keep the raw number values when exporting enum and boolean datasets. Not sure if it's the right choice of if users would prefer enum value labels and `true`/`false`. :shrug: